### ISSUE TITLE
Create Instructions for moving files and folders in Github repository

### ISSUE THREAD

**Issue by tpharrison** [department-of-veterans-affairs/va.gov-team#47331]
## Background
The new platform guidance wants teams to move all research docs into product research directories and rename them to follow a specific format: YYYY-MM-Research-initiative-name.  Making these changes directly in Github is time consuming and tedious.

## Tasks
- [x] Create a recording to demonstrate how to use the Github web-based tool.
- [x] Review with team members

**Comment by tpharrison** [comment]
@andaleliz - I started working on a document that provides instructions for making repo changes in your local environment.  It included some explanation of how Git works so that each instruction made more sense.  It was coming along fine but seemed a bit heavy-handed for someone that just wanted to move some files/folders around.

Fortunately, I stumbled upon a [Github Web-based](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor) editor.  I tried it out and I think it will be much easier to use than working locally.

If it helps, we can do a Zoom walkthrough - it should only take about 5-10 minutes.  If you think it's something that others will find useful, maybe we can open a temporary channel with some documentation pinned to it.  That way if anyone runs into issues using the tool, we can help out.
